src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(unsafe_op_in_unsafe_fn)]
```

Comment:
Just put the unsafe op in an unsafe block?

Reply:
this is b/c i bumped the rust edition. there is...a lot of unsafe in the compress pathway. but, sure